common/workunit/workunit.cpp

```diff
@@ -10149,6 +10149,23 @@ extern WORKUNIT_API bool secDebugWorkunit(const char * wuid, ISecManager &secmgr
     return false;
 }
 
+bool isResultASet(IWUResult *r)
+{
+    SCMStringBuffer schema;
+    r->getResultEclSchema(schema);
+    return !strnicmp(schema.str(), "RECORD set of", strlen("RECORD set of"));
```

Comment:
Hmmmm - looking at the code it won't always output that (if the schema has a name associated, it just outputs the name).

Feels like it would be safer/cleaner/more efficient  to expose CLocalWUResult::getSchema() via IConstWUResult interface and call that.